### PR TITLE
Issue 1667: Improved submission list view performance

### PR DESCRIPTION
**Pull request by wwelling**
Switching the `ManyToMany` join between `NamedSearchFilterGroup` and `SubmissionListColumn` to `LAZY` fetch mode improved submission list view page load time by a factor of 2 when applying filter described in [Using a Saved Filter can cause system to be unresponsive for long periods of time.#1667](https://github.com/TexasDigitalLibrary/Vireo/issues/1667) with small data.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "build": "wvr build --clean"
   },
   "dependencies": {
-    "@wvr/core": "2.2.5-rc.2",
+    "@wvr/core": "https://github.com/TAMULib/Weaver-UI-Core.git#piggy-back-poorly-scoped-requests",
```
Comment (kaladay) [inline package.json:25]: Could you revert this change?
I don't think we should commit github URLs to staging, etc...

**Review by kaladay — CHANGES_REQUESTED** [review]
I've investigated the problem and compared the before and after with this PR.

I believe this does not address the problem.

I can observe the following:

Searching for title, such as "test", results in say 800ms to 1000ms for my particular result set.

Using the saved filter that is the same search as "test" above results in a 2000ms to 3000ms delay.

Using the unsaved filter versus using the saved filter, both with the same search results, should have roughly same performance rather than one being more than twice as slow as the other.

I cannot say that this PR solves the problem described in the referenced issue, however, this PR might still be good as a general performance optimization.

File: package.json (hunk 1 above)
Comment (wwelling) [reply package.json:25]: That version of `@wvr/core` is unrelated to the optimization for the steps your provided. I will gladly remove and wait further review there.

**Review by kaladay — APPROVED** [review]

**Comment by kaladay** [comment]
This message is for reference purposes.

The changes in this PR were re-introduced here: https://github.com/TexasDigitalLibrary/Vireo/pull/1749
(which is now merged.)
